packages/core/src/codewhisperer/service/securityScanHandler.ts

```diff
@@ -133,8 +133,8 @@ export async function pollScanJobStatus(
                 : CodeWhispererConstants.codeScanJobTimeoutSeconds
         if (timer > timeoutSeconds) {
             logger.verbose(`Scan job status: ${status}`)
-            logger.verbose(`Scan job timeout.`)
-            throw new Error('Scan job timeout.')
+            logger.verbose(`Security Scan failed. Amazon Q timed out.`)
+            throw new Error('Security Scan failed. Amazon Q timed out.')
```

Comment:
I think the main catch block already says "Security scan failed.", so this change might cause a duplicate message